apax/cli/templates/train_config_full.yaml

```diff
@@ -39,30 +38,37 @@ data:
 model:
   name: gmnn
   basis:
-    name: gaussian
-    n_basis: 7
-    r_max: 6.0
-    r_min: 0.5
+    name: bessel
+    n_basis: 16
+    r_max: 6.5
```

Comment:
rmax 6.5 is rarely needed in my experience. Most of the time 5-6 is fine. I would suggest 5 as a default.